api_app/analyzers_manager/observable_analyzers/phunter/phunter_base.py

```diff
@@ -0,0 +1,72 @@
+import phonenumbers
+import requests
+from bs4 import BeautifulSoup
+from phonenumbers import carrier
+
+
+class PhunterBase:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def phunt(phone_number: str) -> dict:
+
+        # General Information
+
+        parsed = phonenumbers.parse(phone_number)
```

Comment:
if the number is not valid (that can happen because the observable type is generic so everything is allowed here), what does this library trigger?

This is important because it would be better to properly manage that error here and do not raise an analyzer error because that would make people believe that there is something wrong in the analyzer itself but the problem was the input. In that case, I would prefer to have a valid result of the analyzer with an explicit message saying that only phone numbers are supported